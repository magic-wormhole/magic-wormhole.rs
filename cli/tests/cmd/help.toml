bin.name = "wormhole-rs"
args = "--help"
