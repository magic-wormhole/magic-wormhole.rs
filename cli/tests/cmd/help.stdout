Usage: wormhole-rs[EXE] [OPTIONS] <COMMAND>

Commands:
  send[..][aliases: tx]
  receive[..][aliases: rx]
  send-many[..]
  forward[..]

Options:
  -v, --verbose[..]
  -h, --help[..]
  -V, --version[..]

Run a subcommand with `--help` to know how it's used.
To send files, use `wormhole send <PATH>`.
To receive files, use `wormhole receive <CODE>`.
